libc/string/string_errsig.c

```diff
@@ -175,3 +213,107 @@ TEST_GROUP_RUNNER(string_errsign)
 	RUN_TEST_CASE(string_errsign, strsignal_basic);
 	RUN_TEST_CASE(string_errsign, strsignal_real_time);
 }
+
+
+TEST_GROUP(string_perror);
+
+
+TEST_SETUP(string_perror)
+{
+	errno = 0;
+}
+
+
+TEST_TEAR_DOWN(string_perror)
+{
+}
+
+TEST(string_perror, perror_basic)
+{
+
+	errno = 0;
+	char msg[] = "Some error message";
+	char *res;
+	char exp[100];
+	sprintf(exp, "Some error message: %s\n", strerror(errno));
```

Comment:
"Shouldn't this work the same way?
```suggestion
	char msg[] = "Some error message";
	char *res;
	char exp[100];
	sprintf(expected, "%s: %s\n", msg, strerror(0));
```